lang/en/qtype_questionpy.php

```diff
@@ -63,6 +63,9 @@
 $string['selection_no_icon'] = 'Could not load the icon.';
 $string['selection_required'] = 'Please select a package.';
 
+$string['selection_local_header'] = 'Local Package';
+$string['selection_local_text'] = 'This package was uploaded by a user and might not appear in the package search.';
```

Comment:
Wollen wir es vielleicht lieber "Custom package" nennen? Local ergibt nicht mehr soviel Sinn, da es ja nicht mehr in irgendeiner Liste erscheint -> "lokal" wo?